.gitignore

```diff
@@ -12,3 +12,4 @@ test_output
 wallet_data
 wallet/db
 .idea/
+/node*
```

Comment:
Out of curiosity why?

Reply:
Oh I have a bunch of dirs locally for `node_mainnet`, `node_floonet` etc. And they are all under the root dir of grin...